tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,180 @@
+
+![Moonbase Relay Polkadot JS Apps Home](/images/tutorials/remote-staking-via-xcm/xcm-stake-4.png)
+
+### Building the Destination Multilocation {: #building-the-destination-multilocation }
+
+Let’s get started crafting our XCM message that will transport our remote execution instructions to the Moonbase Alpha parachain to ultimately stake our desired amount of DEV tokens to a chosen collator. To get started, take the following steps: 
+
+1. Unlike the prior steps where the selected account wasn’t relevant, the account selected here must be the account associated with your multilocation derivative account 
+2. Choose the **xcmPallet** pallet
+3. Choose the **send** method
+4. Set the destination version to **V1**
+5. To target Moonbase Alpha, set the destination to:
+
+```
+{
+  "parents":0,
+  "interior":
+    {
+    "x1":
+      {
+      "Parachain": 1000
+    }
+  }
+}
+```
+6. Set the message version to **V2**
+
+![Moonbase Relay Polkadot JS Apps Extrinsics Page](/images/tutorials/remote-staking-via-xcm/xcm-stake-5.png)
+
+In the next section, we’ll start assembling the XCM instructions. 
+
+### Preparing the Structure of the XCM Message {: #preparing-the-structure-of-the-xcm-message }
+
+1. Select **V2** for **XcmVersionedXcm**
+2. Our XCM Message is going to have 3 distinct XCM instructions, so press the first **Add Item** button 3 times 
+3. Below the first XCM Instruction of **WithdrawAsset**, we need to add the asset we’re going to withdraw here, so press the **Add Item** button below **WithdrawAsset** once 
+
+![Preparing the structure of the XCM message](/images/tutorials/remote-staking-via-xcm/xcm-stake-6.png)
+
+### Assembling the Contents of the XCM Message {: #assembling-the-contents-of-the-xcm-message }
+
+Now we’re ready for the fun part! Construct the XCM message that will remotely stake funds on the Moonbase Alpha parachain as follows:
+
+```
+{
+  "WithdrawAsset":
+    [
+      {
+        "id":
+          {
+            "Concrete":
+              {
+                "parents": 0,
+                "interior": {
+                  "X1": {
+                    "PalletInstance": 3
+                  }
+                }
+              }
+            "Fungible": 100000000000000000
+          }
+    ],
+  "BuyExecution":
+    {
+      "fees": {
+        "id":
+          {
+            "Concrete":
+              {
+                "parents": 0,
+                "interior": {
+                  "X1": {
+                    "PalletInstance": 3
+                  }
+                }
+              }
+            "Fungible": 100000000000000000
+          }
+      },
+      "weightLimit": "Unlimited"
+    },
+  "Transact":
+    {
+      "originType": "SovereignAccount",
+      "requiredWeightAtMost": "40000000000",
+      "call": {
+        "encoded": "0x0c113a7d3048f3cb0391bb44b518e5729f07bcc7a45d000064a7b3b6e00d00000000000000002c01000025000000"
+      }
+
+    }
+}
+```
```

Comment:
I think adding the associated encoded call data here would be nice

Reply:
Added!